client/src/components/TVDetailsPage.js

```diff
@@ -9,11 +10,11 @@ const TVDetailsPage = props => {
     reviews: []
```

Comment:
we are now at the point where you can also store `reviews` in its own useState call so that you dont need to constantly maintain `name` and `description` every time you update a specific review (especially on upvote and downvote)